index.html

```diff
@@ -523,6 +531,116 @@ <h4>Workflow Type</h4>
+            <a>Script Events</a> that contain Swedish <a>Text</a> override this by setting
+            <code>xml:lang="sv"</code> on the <code>&lt;p&gt;</code> element.
+            <a>Script Events</a> that contain Danish <a>Text</a> can set the <code>xml:lang</code> attribute
+            or omit it, since the inherited language is the <a>Default Language</a> of the document.
+            In both cases the Script Events' <a>Text</a> objects are <code>&lt;p&gt;</code> elements that represent untranslated
+            content that had an inherent language (in this case dialogue)
+            and therefore set the <a><code>daptm:langSrc</code></a> attribute to their source language,
+            implying that they are in the <a>Original</a> language.</aside>
+        </section>
+
+        <section>
+          <h4>Original Languages</h4>
+          <p>
+            The <dfn data-lt="Original languages list">Original Languages</dfn>,
+            also referred to as "Original languages list",
+            is an optional property of a <a>DAPT Script</a>
+            which represents the languages used in the original content,
+            in other words,
+            the set of languages associated with <a>Text</a> objects in the <a>Original Language Transcript</a>.
+          </p>
+          <p class="note">
+            <a>Text</a> marked as <a>Original</a> can be in a 
+            language not in the <a>Original Languages</a>;
+            this could occur if the original languages list omits languages used for
+            a small proportion of content, for example.
+          </p>
+          <p class="note">
+            The <a>Original Languages</a> are not expected to change after transcription, for example
+            during translation.
+          </p>
+          <p class="note">
+            The <a>Original Languages</a> are generally expected to represent the main languages used in the programme audio.
+            For example this allows the <a>Original Language Transcript</a> to be used as the basis for hard of hearing
+            subtitles. If a programme has significant dialogue in more than one language, for example Danish and Swedish,
+            all those languages used are expected to be included in the
+            <a>Original Languages</a>.
+          </p>
+          <p>The <a>Original Languages</a> are represented in a <a>DAPT Document</a> by the following structure and constraints:</p>
+          <ul>
+            <li>the <dfn><code>daptm:originalLangs</code></dfn> attribute
+              SHOULD be present on the <code>&lt;tt&gt;</code> element
+              and if present its value MUST NOT be empty.</li>
+            <li>the value of the attribute MUST be a set of space separated language identifiers,
+              each being a language tag as defined by [[BCP47]].</li>
+          </ul>
+          <!-- BCP47's syntax does not permit white space within each language tag -->
```

Comment:
There must be a space between `BCP` and `47`, so `BCP 47` instead of `BCP47`. This can be important if people search for `BCP 47`.

Reply:
The system generates a bibliography link in the references section automatically based on this, which is a code from Specref. If they want to follow what the reference points to, they don't need to search for it, they can click on it directly.

The HTML comment is an editorial comment for future potential editors, to remind them why this space separated list is a safe thing to do - I don't think it hugely matters if this has "BCP47" or "BCP 47" but I prefer to have consistency with the reference in the specification.

Reply:
For references, it is accepted to use an acronym (I think spaces are not allowed). In another context, I would prefer the correct spelling of the term because this eases manual and automatic search/detection for/of terms (there is also `BCP-47` in the comment of an expression). But as this only a minor issue I would not insist on a change.

Reply:
I'm going to mark this conversation as resolved for tidiness - @andreastai please unresolve if you think that action is needed.